sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,79 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+
+    defNum = 1
+
+    defLast = False
+    
+    hasPrinted = False
+
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    else:
+        try:
+            defNum = int(args[-1])
+            defLast = True
+        except ValueError:
+            pass
```

Comment:
This part detects whether or not the user input a number as the last element in `args`. It tries to convert it to a string; if the user did in fact use a number, this will return `ValueError` which is ignored with `pass`.